website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -18,6 +18,8 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## April 2025
 
+- **Behavior change**: Users with read-only permissions can now view the [deploy](/docs/deploy/deployments) section (including jobs, runs, and schedules) in a dbt Cloud account.
```

Comment:
it looks like we already added this in the [march release note](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes#march-2025) so i dont think we want to duplicate this?

![Screenshot 2025-04-14 at 10 06 46](https://github.com/user-attachments/assets/64bbcfdb-7996-49e6-a180-375b1c6e6313)


```suggestion
```